internal/cri/server/container_status.go

```diff
@@ -104,6 +107,11 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		st, ft = status.StartedAt, status.FinishedAt
 	}
 
+	runtimeUser, err := toCRIContainerUser(ctx, container)
+	if err != nil {
```

Comment:
Unless we are introducing a new "CRI" requirement here, we should continue to provide status for the container even if there is no task/user.

Reply:
looking at the kublet PR I think this goes with the idea that it is an advisory field.. not required.  Or maybe I'm reading it wrong?

https://github.com/kubernetes/kubernetes/pull/117842/files#diff-c713e8919642d873fdf48fe8fb6d43e5cb2f53fd601066ff53580ea655948f0dR8154

https://github.com/kubernetes/kubernetes/pull/117842/files#r1613656396

Reply:
Let's do a debug log for it.

Reply:
IOW the time to error was when this happened, IMO, not when reporting status or proceeding with a stop/rm.